### ISSUE TITLE
Suggestion: type name capitalization

### ISSUE THREAD

**Issue by dylang** [apollographql/eslint-plugin-graphql#61]
Suggestion:

`TypeNameCap`: `true`

Check that the first letter of types is always capitalized. 

```graphql
# bad
type link {
  href: String
  title: String
}
```

```graphql
# good
type Link {
  href: String
  title: String
}
```

**Comment by jnwng** [comment]
sounds reasonable to me, would happily accept a PR with this functionality!

**Comment by DianaSuvorova** [comment]
@jnwng Can be closed 🙂   Thanks.
